proto/sources.proto

```diff
@@ -270,3 +271,14 @@ message SlackRealtime {
     credentials.SlackTokens tokens = 1;
   }
 }
+
+message Sharepoint {
+  string endpoint = 1 [(validate.rules).string.uri_ref = true];
+  oneof credential {
+    string token = 2;
```

Comment:
refresh_token

Reply:
I actually don't think you need this field at all. refresh_token should be include as part of `credentials.Oauth2`
https://github.com/trufflesecurity/trufflehog/blob/b3d917f9c71789f67e71bed6c3e106a6cbd6d012/proto/credentials.proto#L36-L41